src/client/power-pages/bootstrapdiff/BootstrapDiff.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import * as vscode from 'vscode';
+
+// Green highlight for modifications in the output file
+const outputDecorationType = vscode.window.createTextEditorDecorationType({
+	backgroundColor: '#90ee90'
+});
+
+// Red highlight for the classes output in the input file
+const inputDecorationType = vscode.window.createTextEditorDecorationType({
+	backgroundColor: '#E1AEAD'
+});
+
+// object for storing class range
+type ClassRange = {
+	start: number;
+	end : number;
+	offset : number;
+	matchedClass : string;
+	replacedClass: string;
+	message: string;
+};
+
+let matchedClasses : ClassRange[][];
+
+export async function bootstrapDiff()
+{
+    const editor = vscode.window.activeTextEditor;
+		if(!editor)
+		{
+			// Handle this to prompt the user to open a file and the run the command
+			return;
+		}
+
+		const inputPath = editor.document.uri.path;
+
+		const diffFileName = inputPath + "-diff.json";
+		let diffData = "";
+		await vscode.workspace.openTextDocument(vscode.Uri.file(diffFileName)).then((document) => {
+			diffData = document.getText();
+		});
+
+		matchedClasses = JSON.parse(diffData);
+		hihglightReplacedClasses(editor);
+
+		const workspaceFolders = vscode.workspace?.workspaceFolders;
+		if(!workspaceFolders)
+		{
+			// Handle open the website folder in vscode
+			return;
+		}
+
+		const websiteFolder = workspaceFolders[0].name;
```

Comment:
Verify if this would work fine with multi-root workspaces.
https://github.com/microsoft/vscode/wiki/Adopting-Multi-Root-Workspace-APIs

Reply:
Fixed this issue now by picking folder name using the open documents uri